java/runtime-native/src/main/java/org/ray/runner/RunManager.java

```diff
@@ -852,6 +853,49 @@ private AddressInfo startObjectManager(int index, AddressInfo info, String workD
     }
   }
 
+  private Map<String, Double> getResourcesFromString(String resources) {
+    Map<String, Double> ret = new HashMap<>();
+    if (resources != null) {
+      String[] items = resources.split(",");
+      for (String item : items) {
+        String trimItem = item.trim();
+        String[] resourcePair = trimItem.split(":");
+
+        if (resourcePair.length != 2) {
+          RayLog.core.error("Static resources configure format error.");
```

Comment:
just `throw new InvalidArgumentException("Format of static resources configure is invalid")`